frida-sys/build.rs

```diff
@@ -47,4 +63,14 @@ fn main() {
     bindings
         .write_to_file(out_path.join("bindings.rs"))
         .unwrap();
+
+    #[cfg(windows)]
+    &[
+        "dnsapi", "iphlpapi", "psapi", "winmm", "ws2_32", "advapi32", "crypt32", "gdi32",
```

Comment:
FWIW @meme this is from
https://github.com/frida/frida/blob/02cfb5dfeda4d2aff68fb7b71e49e86944f9e5b6/releng/devkit.py#L128
in case you would want to add a note here